temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
**CODE_QUALITY**: The `pdb` import statement and `print` statement seem to be for debugging purposes. These should be removed before merging the code to the main branch.
```python
# Remove the following lines before merging:
# import pdb
# print("hello")
```